Formula/encfs.rb

```diff
@@ -23,6 +23,8 @@ class Encfs < Formula
 
   def install
     ENV.cxx11
+    ENV.append "CMAKE_INCLUDE_PATH", "/usr/local/include", ":"
```

Comment:
Could we instead pass `-DFUSE_INCLUDE_DIR=/usr/local` to `cmake`?

Reply:
Done